pkg/resources/kubernetes-dashboard/deployment.go

```diff
@@ -166,3 +177,12 @@ func getVolumes() []corev1.Volume {
 		},
 	}
 }
+
+func getDashboardVersion(clusterVersion semver.Semver) (string, error) {
+	// check the GitHub releases for find compat info on the dashboard:
+	// https://github.com/kubernetes/dashboard/releases
+
+	// v2.4.0 is the latest version and supports Kubernetes 1.20/1.21;
+	// there is no official release to support 1.22+ yet.
+	return "v2.4.0", nil
```

Comment:
@xrstf FYI: We have released yesterday `v2.5.0`. It can be used for the `v1.22` and `v1.23` Kubernetes clusters.

Reply:
https://github.com/kubernetes/dashboard/releases/tag/v2.5.0 says it's for 1.23 only. Which one is right? ^^